CHANGELOG_UNRELEASED.md

```diff
@@ -207,6 +207,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     * new lemmas: `eigenpolyP`, `eigenvalue_poly`, `eigenspace_sub_geigen`,
   + new `map_mx` lemmas: `map_kermxpoly`, `map_geigenspace`, `eigenpoly_map`.
 
+- in `ssralg.v`: 
+   + Lemma `expr_sum` : equivalent for ring of `expn_sum`
+   + Lemma `prodr_natmul` : generalization of `prodrMn_const`.
+   Its name will become `prodrMn` in the next release when this name will become available (cf. Renamed section)
```

Comment:
Did we reach a consensus on this 2 step renamings/deprecations? @CohenCyril

Reply:
Yes I think we did!